.gitignore

```diff
@@ -27,3 +27,8 @@ wasm/*.wasm
 
 # app version string file
 apps/version.rs
+
+# Governance artifacts
```

Comment:
let's not do this - any files created by test(s) should be inside a temp dir

Reply:
Yes, it's actually what we are now doing with 147cf381fc9fef7a05e03ddfff1de031aee99f4c, I'll revert the gitignore, thanks!